spec/features/memes/user_sees_memes_for_category_spec.rb

```diff
@@ -11,11 +11,11 @@
     expect(page).to have_content ek.name
     expect(page).to have_content pizza.name
     expect(page).to have_xpath("//img[contains(@src,pizza.img_url)]")
-    expect(page).to have_content pizza.user.email
+    expect(page).to have_content pizza.user.username
   end
 
```

Comment:
This is fine, but a quick caveat for future tests: Even though we discourage hard-coding in your actual code, it's actually _better_, in some cases, to hard-code your test expectations. That way, you can be absolutely sure your tests are working as you expect, i.e. returning a specific value! So, in this case, `expect(page).to have_content "bob"` is a bit better than what you've written above.